stactools_pipelines/pipelines/icesat2_boreal/config.yaml

```diff
@@ -0,0 +1,8 @@
+---
+  id: "icesat2_boreal"
+  compute: "awslambda"
+  secret_arn: "arn:aws:secretsmanager:us-west-2:916098889494:secret:MAAP-STAC-auth-dev/MAAP-workflows-EsykqB" # cognito secret ARN
+  ingestor_url: "https://stac-ingestor.dit.maap-project.org"
+  inventory_location: "s3://maap-ops-workspace/shared/henrydevseed/icesat2-boreal-v2.1-inventory.csv"
```

Comment:
any issues with creating the inventory csv? I had an awscli command I could share

Reply:
This worked fine! I had an inventory csv from Paul et al and just had to reshape it to make this file. Either way I am curious about the awscli command :eyes:

Reply:
More of an echo than a awscli command; it seems a bit janky but it was useful at the time - 

> (echo "s3_path"; aws s3 ls s3://nasa-maap-data-store/file-staging/nasa-map/GEDI_CalVal_Lidar_Data___2/ --recursive | grep ".las$" | awk '{print "s3://nasa-maap-data-store/" $4}') > GEDI_CalVal_Lidar_Data.csv